resources/autoload.php

```diff
@@ -18,7 +18,7 @@
 require_once __DIR__ . "/lib/UnitySite.php";
 require_once __DIR__ . "/lib/UnityConfig.php";
 require_once __DIR__ . "/lib/UnityWebhook.php";
-require_once __DIR__ . "/lib/UnityRedis.php";
+include_once __DIR__ . "/lib/UnityRedis.php";
```

Comment:
This can be left as a require, as the file will always be there, it's just that it won't always be used